contribute/style-guides/e2e.md

```diff
@@ -0,0 +1,140 @@
+# End to End test framework
+As our company grows our need for end to end (called e2e from now on) tests have also grown. Today we're using a minimal home grown solution built on top of Cypress for our e2e tests. 
+
+## Basic concepts
+A good introduction to some good e2e practices can be found [here](https://martinfowler.com/bliki/PageObject.html).
+- `Selector`, an unique identifier that is used from the e2e framework to retrieve an element from the Browser 
+- `Page`, an abstraction for an object that contains one or more `Selectors`
+- `Flow`, an abstraction that contains a sequence of actions on one or more `Pages` that can be reused and shared between tests
+
+## Basic example
+Lets start with a simple example with a single selector and for simplicity all examples will be in JSX.
+
+In our example app we have an input that we during our e2e test want to type some text in.
```

Comment:
```suggestion
In our example app we have an input that we during our e2e test want to type some text into.
```

Reply:
In our example app, we have an input that we want to type some text into during our e2e test.